index.html

```diff
@@ -39,6 +39,125 @@ <h2 class="lets-connect">LET’S CONNECT</h2>
+                </ul>
+                <button class="card-button" type="button">See Project</button>
+            </div>
+
+            <div class="card-item">
+                <img src="./images/cards/card4.svg" alt="Card 4 Screenshot">
+                <h2 class="card-heading">Multi-Post Stories</h2>
+                <div>
+                    <span class="canopy">CANOPY</span>
+                    <span class="gray-matter">• Back End Dev • 2015</span>
+                </div>
+                <p class="card-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</p>
+                <ul class="card-list">
+                    <li class="card-list-items">html</li>
+                    <li class="card-list-items">css</li>
+                    <li class="card-list-items">javascript</li>
+                </ul>
+                <button class="card-button" type="button">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-me">
+        <div class="about-me-content">
+            <h1 class="about-me-heading">About Myself</h1>
+
+            <p class="about-me-paragraph">Hello I’m a software developer! I can help you build a product , feature or
+                website Look through some of my work and experience! If you like what you see and have a project you
+                need coded, don’t hestiate to contact me.</p>
+
+            <h2 class="abt-me-connect">LET’S CONNECT</h2>
+
+            <ul class="about-me-icons">
+                <li><a href="#" class="icon-link"><img src="./images/icons/github.svg" alt="Github Logo"></a></li>
+                <li><a href="#" class="icon-link"><img src="./images/icons/linkedin.svg" alt="Linkedin Logo"></a></li>
+                <li><a href="#" class="icon-link"><img src="./images/icons/finger.svg" alt="Two Fingers"></a></li>
+                <li><a href="#" class="icon-link"><img src="./images/icons/twitter.svg" alt="Twitter Logo"></a></li>
+                <li><a href="#" class="icon-link"><img src="./images/icons/medium.svg" alt="Medium Logo"></a></li>
+            </ul>
+
+            <button class="resume-btn" type="button">Get my resume</button>
+
+            <div class="skills">
+                <div class="languages">
+                    <h3 class="skill-text-style">Language</h3>
+                    <img src="./images/icons/Union.svg" alt="open tag">
+                </div>
+                <ul class="language-icons">
+                    <li class="skill-icons"><img src="./images/icons/Javascript.svg" alt=""><span class="language-text">Javascript</span>
+                    </li>
+                    <li class="skill-icons"><img src="./images/icons/HTML.svg" alt=""><span class="language-text">HTML</span></li>
+                    <li class="skill-icons"><img src="./images/icons/CSS.svg" alt=""><span class="language-text">CSS</span></li>
+                </ul>

```

Comment:
Great job implementing this section of the portfolio, however, if you look closely, the languages are not properly aligned in line with the Figma template, and your design is not starting from the `Languages` header.

- Figma Design
![image](https://user-images.githubusercontent.com/81932478/164513661-6f87e67a-47ca-4dfd-9bc8-9761a3aa5a90.png)

- Your Design
![image](https://user-images.githubusercontent.com/81932478/164513840-d883208c-8efd-4fae-95cf-ed759f19cd2c.png)